### PR TITLE
add simulacrum, centril to compiler-contributors

### PR DESCRIPTION
**Pull request by nikomatsakis**
Add @Mark-Simulacrum and @Centril to the "compiler contributors" team, both of whom have contributing more and more to the compiler lately. A big ❤️ thank you ❤️ to both of them! 🎉

### CODE REVIEW

File: teams/compiler-contributors.toml [hunk 1]
```diff
@@ -12,6 +13,7 @@ members = [
   "nikic",
   "nnethercote",
   "scalexm",
+  "Mark-Simulacrum",
```
Comment (spastorino) [inline teams/compiler-contributors.toml]: Seems like your intention was to alphabetically sort names, but given that's not simulacrum but Mark-Simulacrum should be placed a bit above :smile:

File: teams/compiler-contributors.toml [hunk 2]
```diff
@@ -4,6 +4,7 @@ subteam-of = "compiler"
 [people]
 leads = []
 members = [
+  "centril",
```
Comment (JohnTitor) [inline teams/compiler-contributors.toml]:
```suggestion
  "Centril",
```

File: teams/compiler-contributors.toml (hunk 1 above)
Comment (nikomatsakis) [reply teams/compiler-contributors.toml]: ha thanks, I originally had him in there "simulacrum"...

Comment (spastorino) [reply teams/compiler-contributors.toml]: Yeah, I guessed that that was the issue :).

**Pull request merged**